.github/workflows/run-batch.yml

```
@@ -0,0 +1,45 @@
+name: Batch CodeDeploy CI
+
+on:
+  workflow_dispatch:
```

I would expect we want this to run on semantically versioned tags:

```suggestion
on:
  push:
    tags: [ v*.*.* ]
  workflow_dispatch:
```